README.md

```diff
@@ -1,30 +1,29 @@
 # Robot Framework katas
 
+In this series of exercises we're going to run several automated tests against web application.
+
 ## Requirements
 
-In this series of exercises we're going to run several automated tests against web application. To do
-that we're going to need to have the following items installed:
+To be able to run the exercises the following items need to be installed:
 
-- Python (version 3.7 or newer): https://www.python.org/downloads/
+- Python (version 3.8 or newer): https://www.python.org/downloads/
```

Comment:
Should we also skip the 3.8 since EOL of 3.8 is already this year october https://devguide.python.org/versions/

Reply:
I just mirrored the browser library requirements. Other than that I don't think we have a reason to have a newer python

Reply:
And I'm fine with updating it again in 6 months